### DISCUSSION TITLE
How to get the corpus used by OSS-fuzz for a given target ?

### DISCUSSION THREAD

**Discussion by sultah** [google/oss-fuzz#12543] [Q&A] [answered]
Hi,

Is there a way to get the corpus used by OSS-fuzz for a given target ? I cannot find anything explaining how to get the corpus

Thanks,

Hassan

**Comment by maflcko** [comment, marked as answer]
The public ones should be located at `f"https://storage.googleapis.com/{project}-backup.clusterfuzz-external.appspot.com/corpus/libFuzzer/{project}_{target}/public.zip"`

See also https://google.github.io/oss-fuzz/advanced-topics/corpora/#corpus-backups

**Comment by sultah** [comment]
Weird, I swear this worked a few days ago, but now if I enter for example :

gsutil -m cp -r gs://storage.googleapis.com/libpng-backup.clusterfuzz-external.appspot.com/corpus/libFuzzer/libpng_read_fuzzer/public.zip /tmp

I get a 404 bucket not found exception. Were the buckets removed / renamed ?

**Comment by maflcko** [comment]
With gsutil you may have to log in using  the correct google account, but I haven't tried this.

Locally, this works just fine:

```
curl -v -L -O 'https://storage.googleapis.com/libpng-backup.clusterfuzz-external.appspot.com/corpus/libFuzzer/libpng_read_fuzzer/public.zip'

**Reply by catenacyber** [reply]
Looks like it does not work anymore today
